pkg/virt-api/webhooks/validating-webhook/admitters/flavor-admitter.go

```diff
@@ -26,42 +25,68 @@ func (f *FlavorAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissionv1.Adm
 			Version:  flavorv1alpha1.SchemeGroupVersion.Version,
 			Resource: flavor.PluralResourceName,
 		},
-		func(raw []byte) ([]flavorv1alpha1.VirtualMachineFlavorProfile, error) {
-			flavorObj := flavorv1alpha1.VirtualMachineFlavor{}
+		func(raw []byte) (*flavorv1alpha1.VirtualMachineFlavor, error) {
```

Comment:
This function can return `VirtualMachineFlavorSpec` and then the webhook can use common code for Flavor and ClusterFlavor, similarly to what was done with profiles.

Reply:
Yeah I could do that but I think I wanted to keep a reference to the overall flavorobj on the off chance we need to validate anything outside of the spec in the future. FWIW I add some validation later on using the spec here:

https://github.com/kubevirt/kubevirt/pull/7578/commits/9dd3454726d45fd0ad9e92e7363d4278986f3f18